solutions/standard/variables.tf

```diff
@@ -359,12 +359,24 @@ variable "existing_code_engine_project_id" {
 
 variable "enable_kibana_dashboard" {
   type        = bool
-  description = "Set it true to deploy Kibana in code engine. NOTE: Kibana image is coming direcly from the official registry (https://www.docker.elastic.co/) and not certified by the IBM."
+  description = "Set to true to deploy Kibana in Code Engine. NOTE: By default, the Kibana image will be pulled from the official Elastic registry (docker.elastic.co) and is not certified by IBM, however this can be overridden using the `kibana_image_reference` input."
```

Comment:
`kibana_image_reference` no longer exists. You need to reference `kibana_registry_namespace_image` and `kibana_image_digest`

Reply:
Done.